frontend/src/hooks/index.tsx

```diff
@@ -29,7 +29,6 @@ export {
   usePublicDashboard,
   useDashboards,
   useWidget,
-  useColors,
```

Comment:
I wonder if it was necessary to get rid of the `userColors` hook? I understand that the colors are now coming from the Settings object but you could have used the `useSettings` inside the `useColors` hook to avoid such a big refactor.